### ISSUE TITLE
Windows 2019 Builds on GCE are failing.

### ISSUE THREAD

**Issue by upodroid** [hashicorp/packer#8173]
### Overview of the Issue

Instance fails mysteriously.

#### Reproduction Steps

Run packer with the manifest below.

### Packer version

v1.4.3

### Simplified Packer Buildfile

```
 "builders": [
    {
      "type": "googlecompute",
      "region": "{{user `region`}}",
      "zone": "{{user `zone`}}",
      "project_id" : "{{user `project_id`}}",
      "communicator": "winrm",
      "winrm_username": "packer_user",
      "winrm_insecure": true,
      "winrm_use_ssl": true,
      "source_image_family": "windows-2019",
      "preemptible": "true",
      "image_description": "Modified Base Windows Server 2019",
      "disk_type": "pd-ssd",
      "image_labels": {"enviroment":"dev"},
      "disk_size": "50",
      "image_name": "build-image-windows2019-{{isotime | clean_image_name}}",
      "image_family": "windows2019-base",
      "scopes": [
        "https://www.googleapis.com/auth/cloud-platform"
      ],
      "metadata": {
        "install-stackdriver-agent": true,
        "install-stackdriver-logging" : true,
        "install-default-packages": true,
        "install-cloud-sql-proxy": true,
        "windows-startup-script-cmd": "winrm quickconfig -quiet & net user /add packer_user & net localgroup administrators packer_user /add & winrm set winrm/config/service/auth @{Basic=\"true\"}"
    }
  }
],

"provisioners": [
    
  {
    "type": "powershell",
    "script": "rs-config.ps1"
  }
],
```

### Operating system and Environment details

Gitlab CI Job.

I'm using this docker image: quay.io/upodroid/build-image
https://github.com/borg-land/build-image

### Log Fragments and crash.log files
```
$ packer build windows2019.json
googlecompute output will be in this color.

==> googlecompute: Checking image does not exist...
==> googlecompute: Creating temporary SSH key for instance...
==> googlecompute: Using image: windows-server-2019-dc-v20190910
==> googlecompute: Creating instance...
    googlecompute: Loading zone: europe-west2-b
    googlecompute: Loading machine type: n1-standard-1
    googlecompute: Requesting instance creation...
    googlecompute: Waiting for creation operation to complete...
    googlecompute: Instance has been created!
==> googlecompute: Creating windows user for instance...
==> googlecompute: Deleting instance...
    googlecompute: Instance has been deleted!
==> googlecompute: Deleting disk...
    googlecompute: Disk has been deleted!
Build 'googlecompute' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> googlecompute: unexpected EOF

==> Builds finished but no artifacts were created.
panic: runtime error: invalid memory address or nil pointer dereference
2019/09/30 12:50:45 packer: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1cc79f7]
2019/09/30 12:50:45 packer: 
2019/09/30 12:50:45 packer: goroutine 95 [running]:
2019/09/30 12:50:45 packer: github.com/hashicorp/packer/builder/googlecompute.(*StepCreateWindowsPassword).Run(0xc00034be40, 0x3fadba0, 0xc00033ba80, 0x3f936a0, 0xc000038870, 0x0)
2019/09/30 12:50:45 packer: 	/Users/mmarsh/Projects/packer/builder/googlecompute/step_create_windows_password.go:63 +0x657
2019/09/30 12:50:45 packer: github.com/hashicorp/packer/helper/multistep.(*BasicRunner).Run(0xc000038900, 0x3fadba0, 0xc00033ba80, 0x3f936a0, 0xc000038870)
2019/09/30 12:50:45 packer: 	/Users/mmarsh/Projects/packer/helper/multistep/basic_runner.go:67 +0x1ec
2019/09/30 12:50:45 packer: github.com/hashicorp/packer/builder/googlecompute.(*Builder).Run(0xc000412060, 0x3fadba0, 0xc00033ba80, 0x3fc97a0, 0xc00034bd20, 0x3f46f40, 0xc00039d180, 0x40b9b9, 0xc000038660, 0x30, ...)
2019/09/30 12:50:45 packer: 	/Users/mmarsh/Projects/packer/builder/googlecompute/builder.go:86 +0x95a
2019/09/30 12:50:45 packer: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Run(0xc0002d0ed0, 0x1, 0xc000445200, 0x0, 0x0)
2019/09/30 12:50:45 packer: 	/Users/mmarsh/Projects/packer/packer/rpc/builder.go:109 +0x231
2019/09/30 12:50:45 packer: reflect.Value.call(0xc0003f6900, 0xc00000e170, 0x13, 0x3998dd5, 0x4, 0xc0003b8f18, 0x3, 0x3, 0xc000388340, 0x3, ...)
2019/09/30 12:50:45 packer: 	/usr/local/go/src/reflect/value.go:447 +0x461
2019/09/30 12:50:45 packer: reflect.Value.Call(0xc0003f6900, 0xc00000e170, 0x13, 0xc000018718, 0x3, 0x3, 0xc000062540, 0xc000018730, 0x404e35)
2019/09/30 12:50:45 packer: 	/usr/local/go/src/reflect/value.go:308 +0xa4
2019/09/30 12:50:45 packer: net/rpc.(*service).call(0xc0003882c0, 0xc00042a0a0, 0xc000444590, 0xc0004445a0, 0xc0000fa780, 0xc000413660, 0x2ecd180, 0xc0004451fc, 0x18a, 0x2e1fce0, ...)
2019/09/30 12:50:45 packer: 	/usr/local/go/src/net/rpc/server.go:384 +0x14e
2019/09/30 12:50:45 packer: created by net/rpc.(*Server).ServeCodec
2019/09/30 12:50:45 packer: 	/usr/local/go/src/net/rpc/server.go:481 +0x42b
2019/09/30 12:50:45 [INFO] (telemetry) ending googlecompute
2019/09/30 12:50:45 ui error: Build 'googlecompute' errored: unexpected EOF
2019/09/30 12:50:45 machine readable: error-count []string{"1"}
2019/09/30 12:50:45 ui error: 
==> Some builds didn't complete successfully and had errors:
2019/09/30 12:50:45 machine readable: googlecompute,error []string{"unexpected EOF"}
2019/09/30 12:50:45 ui error: --> googlecompute: unexpected EOF
2019/09/30 12:50:45 ui: 
==> Builds finished but no artifacts were created.
2019/09/30 12:50:45 [INFO] (telemetry) Finalizing.
2019/09/30 12:50:45 /usr/local/bin/packer: plugin process exited
2019/09/30 12:50:45 waiting for all plugin processes to complete...
2019/09/30 12:50:45 /usr/local/bin/packer: plugin process exited
2019/09/30 12:50:45 /usr/local/bin/packer: plugin process exited



!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Packer crashed! This is always indicative of a bug within Packer.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Packer[1] so that we can fix this.

[1]: https://github.com/hashicorp/packer/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
ERROR: Job failed: command terminated with exit code 1
```

**Comment by upodroid** [comment]
![image](https://user-images.githubusercontent.com/25100905/65884503-e21f2b00-e390-11e9-8282-1b3cf458b25a.png)

Serial Console output from Google Cloud

**Comment by SwampDragons** [comment]
Hi, thanks for reaching out. As you noted, this is a duplicate. I'll leave this open until we release v1.4.4, which is probably going to happen tomorrow.

**Comment by upodroid** [comment]
Thank you

**Comment by SwampDragons** [comment]
Closed in the 1.4.4 release
